src/utils/events.js

```diff
@@ -11,6 +11,7 @@ export function convertStandardToMilitaryTime(standardTime) {
 	let time = standardTime.split(":");
```

Comment:
This should be destructured to show what time is what.
```suggestion
	let [hour,minute,period] = standardTime.split(":");
```
If period is not used then you do not need to add it.